scripts/cloud-api-demo/renew_certificates.py

```diff
@@ -0,0 +1,27 @@
+import argparse
+import subprocess
+
+def execute_command(cmd):
+    """Execute a shell command."""
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    output, error = process.communicate()
+
+    if process.returncode != 0:
+        print(f"Error occurred: {error.decode().strip()}")
```

Comment:
If there is an error it must immediately exit with a non zero exit code after printing the message

Reply:
Implemented